chapters/pt/chapter1/10.mdx

```diff
@@ -0,0 +1,68 @@
+# Questionário de fim de capítulo
+
+Este capítulo cobriu muito terreno! Não se preocupe se você não entendeu todos os detalhes; os próximos capítulos o ajudarão a entender como as coisas funcionam debaixo do capô.
+
+Primeiro, porém, vamos testar o que você aprendeu neste capítulo!
+
+### [](https://github.com/huggingface/course/blob/main/chapters/en/chapter1/10.mdx#1-explore-the-hub-and-look-for-the-roberta-large-mnli-checkpoint-what-task-does-it-perform)1. Explore o Hub e olhe para o checkpoint `roberta-large-mnli` . Que tarefa ele executa?
```

Comment:
What is the reason for these links in each question?

Reply:
I think you copy the file from GitHub viewer, and it broke all the links in this file